src/styles/challenges.scss

```diff
@@ -67,5 +67,9 @@
     justify-content: flex-end;
     width: 23vw;
     height: auto;
+
+    img {
+      width: 150%;
```

Comment:
This feels a little hacky but it only applies to the one image (the other ones are SVG) and I checked that it works on every screen size.

Reply:
The logo was such so much wider than the other images that the old styles were making it way too big